solarforecastarbiter/reports/templates/metrics_hour.md

```diff
@@ -0,0 +1,8 @@
+
+### Hourly analysis
+
+Metrics for each hour (0-23) of the analysis period are displayed in tables and figures below.
```

Comment:
we'd want 1-24 for interval labeling = 'ending', but that's probably best addressed in another pull request. If you agree, then let's make an issue for it.

Reply:
Yeah. Shouldn't be too difficult but I agree I'd prefer to handle in a smaller PR.  I can open an issue for it.

Reply:
Opened #234